bin/letsencrypt-create-wildcard

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+#
+# Create a wildcard certificate from letsencrypt
+#
+
+export MOON_FILE=false
+
+source ${MOON_SHELL}
+
+if [[ $# -lt 1 ]]; then
+    echoerr "Usage: $(basename $0) DOMAIN_NAME [ACME_ARGS]"
+    echoerr
+    echoerr "Note: Specify the base domain name for the certificate; this"
+    echoerr "      script adds both '\$domain' and '*.\$domain'."
+    echoerr
+    echoerr "Example:"
+    echoerr "    $ $(basename $0) foo.example.com"
+    exit 0
+else
+    DOMAIN_NAME=$1
+    shift
+    ACME_ARGS="$*"
+fi
+
+KEY_BITS="4096"
+
+clean_exit () {
+    [[ $# == 0 ]] \
+        && local retr=0 \
+        || local retr=$1
+    local file
+
+    echoerr "INFO: Cleaning up temporary files"
+    for file in ${CERT_FILE} ${KEY_FILE} ${CA_FILE}; do
+        if [[ -f ${file} ]]; then
+            echoerr "INFO: Removing '${file}'"
+            rm -f ${file}
+        else
+            echoerr "INFO: Skipping non-existent file '${file}'"
+        fi
+    done
+
+    exit ${retr}
+}
+
+# Moonshell has a gitignore for this directory
+pushd $(dirname $0)/../ >/dev/null
+    if [[ ! -d "${HOME}/.acme.sh/${DOMAIN_NAME}" ]]; then
+        echoerr "WARNING: You have not administered this domain before"
+    elif [[ ! -f "${HOME}/.acme.sh/${DOMAIN_NAME}/${DOMAIN_NAME}.key" ]]; then
+        echoerr "WARNING: You are missing the domain's key '${HOME}/.acme.sh/${DOMAIN_NAME}/${DOMAIN_NAME}.key'"
+    else
+        echoerr "INFO: Found domain key '${HOME}/.acme.sh/${DOMAIN_NAME}/${DOMAIN_NAME}.key'"
+    fi
+
+    echoerr "INFO: Creating temporary files"
+    CERT_FILE=$(mktemp)
+    KEY_FILE=$(mktemp)
+    CA_FILE=$(mktemp)
+
+    echoerr "INFO: CERT_FILE=${CERT_FILE}"
+    echoerr "INFO: KEY_FILE=${KEY_FILE}"
+    echoerr "INFO: CA_FILE=${CA_FILE}"
+
+    if [[ ! -f "./acme.sh/acme.sh" ]]; then
```

Comment:
Need to clone acme.sh into a writable directory, not moonshell root.

Reply:
Addressed in recent commit. I'm using a pushd to $MOON_VAR, so `./` should be well writable.. should..